src/lib/components/AccordionItem.svelte

```diff
@@ -18,24 +18,37 @@
 	const handleClick = () => (open = !open);
 </script>
 
-<button
-	class="accordion-button {buttonClasses}"
-	{disabled}
-	data-cy-id={buttonTestId}
-	on:click={(e) => {
-		if ($$slots.body && showBody) {
-			handleClick();
-			clickLogic ? clickLogic(e) : null;
-		}
-	}}
->
-	{#if (showBody && $$slots.body)}
-		<div class="chevron">
-			<Icon height={17} width={17} iconSVG={CHEVRON_SVG} direction={open ? Direction.Up : Direction.Down}/>
-		</div>
-	{/if}
-	<slot name="button" />
-</button>
+<div class={disabled ? 'not-allowed' : ''}>
+	<!-- svelte-ignore a11y-click-events-have-key-events -->
+	<!-- svelte-ignore a11y-interactive-supports-focus -->
+	<div
```

Comment:
This was also the initial implementation. The problem with the `<button>` tag is that the user cannot copy the text inside it, and we have the use case where we want to copy different values that are inside a row in a table. This feedback came directly from Andreas & Dirk. This is why I went with a `<div>` tag here, but with `role="button"` set, so that we can stay a11y compliant.